src/parser/transform/constraint/transform_constraint.cpp

```diff
@@ -71,6 +71,11 @@ unique_ptr<Constraint> Transformer::TransformConstraint(duckdb_libpgquery::PGLis
 	case duckdb_libpgquery::PG_CONSTR_NULL:
 		return nullptr;
 	case duckdb_libpgquery::PG_CONSTR_DEFAULT:
+		if (column.default_value) {
+			// A default value may already exist for the column if, for example,
+			// the column was declared with a SERIAL pseudotype
+			throw ParserException("Duplicate default value for column");
```

Comment:
(This is the exception you get from Postgres when you try to specify a default value for a SERIAL column type)